cookbooks/new_ping.md

```diff
@@ -53,65 +63,115 @@ field. The `schema` element has top-level fields (e.g. `Timestamp`, `Type`), as
-
-Work is happening to make a
-[generic endpoint](https://bugzilla.mozilla.org/show_bug.cgi?id=1363160).
-These pings can be easily registered and sent to our servers and will
-be automatically available in Re:dash. Please check back later for those docs.
+[iOS Telemetry APIs](https://github.com/mozilla-mobile/telemetry-ios). 
+
+For non-Telemetry data, see [our HTTP edge server specification](/concepts/pipeline/http_edge_spec.md)
+and specifically the [non-Telemetry example](/concepts/pipeline/http_edge_spec.md#postput-request) for the expected format. The edge
+server endpoint is `https://incoming.telemetry.mozilla.org`.
+
+(Non-Telemetry) Access Your Data
+--------------------------------
+First confirm with the reviewers of
+[your schema pull request](#submit-schema-to-mozilla-servicesmozilla-pipeline-schemas)
+that your schemas have been deployed.
+
+In the following links, replace `<namespace>`, `<doctype>` And `<docversion>` with
+appropriate values. Also replace `-` with `_` in `<namespace>` if your
+namespace contains `-` characters.
+
+### CEP
+Once you've sent some pings, refer to the following real-time analysis plugins
+to verify that your data is being processed:
+
+- `https://pipeline-cep.prod.mozaws.net/dashboard_output/graphs/analysis.moz_generic_error_monitor.<namespace>.html`
+- `https://pipeline-cep.prod.mozaws.net/dashboard_output/analysis.moz_generic_<namespace>_<doctype>_<docversion>.submissions.json`
+- `https://pipeline-cep.prod.mozaws.net/dashboard_output/analysis.moz_generic_<namespace>_<doctype>_<docversion>.errors.txt`
+
+If this first graph shows ingestion errors, you can view the corresponding
+error messages in the third link. Otherwise, you should be able to view the
+last ten processed submissions via the second link. You can also write your own
+custom real-time analysis plugins using this same infrastructure if you desire;
+use the above plugins as examples and see
+[here](/cookbooks/realtime_analysis_plugin.md) for a more detailed explanation.
+
+If you encounter schema validation errors, you can fix your data or
+[submit another pull request](#submit-schema-to-mozilla-servicesmozilla-pipeline-schemas)
+to amend your schemas. Backwards-incompatible schema changes should generally
+be accompanied by an increment to `docversion`.
+
+Once you've established that your pings are flowing through the real-time
+system, verify that you can access the data from the downstream systems.
+
+### STMO
+In the Athena data source, a new table
+`<namespace>_<doctype>_parquet_<docversion>` will be created for your data. A
+convenience pointer `<namespace>_<doctype>_parquet` will also refer to the latest
+available `docversion` of the ping. The data is partitioned by
+`submission_date_s3` which is formatted as `%Y%m%d`, like `20180130`. Refer to the 
+[STMO documentation](/tools/stmo.md) for general information about using Re:dash.
+
+This table may take up to a day to appear in the Athena source; if you still
```

Comment:
I think it'd be good to clarify that when it does appear, it will contain all the pings sent during that first day.